homeassistant/helpers/config_validation.py

```diff
@@ -267,6 +297,19 @@ def url(value: Any) -> str:
     raise vol.Invalid('invalid url')
 
 
+def email(value: Any) -> str:
```

Comment:
Ok, if that is really required. But I think your RFC reference is more correct and you need `.local`